test/organizations/data/OrganizationCompletionEntity.js

```diff
@@ -0,0 +1,233 @@
+export const testData = {
+	trackingNoDisplay: {
+		'class': [
+			'active',
+			'course-offering',
+			'completion-tracked'
+		],
+		'properties': {
+			'name': 'Course Name',
+			'code': 'SCI100',
+			'startDate': '2100-01-01T00:00:00.000Z',
+			'endDate': null,
+			'isActive': false
+		},
+		'entities': [
+			{
+				'class': [
+					'course-image'
+				],
+				'rel': [
+					'https://api.brightspace.com/rels/organization-image'
+				],
+				'href': '../data/image.json'
+			},
+			{
+				'class': [
+					'relatvie-uri'
+				],
+				'rel': [
+					'https://api.brightspace.com/rels/organization-image'
+				]
+			}
+		],
+		'links': [
+			{
+				'rel': [
+					'self'
+				],
+				'href': '../data/organization-future.json'
+			},
+			{
+				'rel': [
+					'https://api.brightspace.com/rels/parent-semester'
+				],
+				'href': '../data/semester.json'
+			}
+		],
+		'actions': [
+			{
+				'href': '../completion/tracking.json',
```

Comment:
Does this file `../completion/tracking.json` need to exist?

Reply:
No. I stole the info from the organizations repo. Since we never actually check the href, I figured this was fine. It can be changed if desired.